LICENSE

```diff
@@ -19,7 +19,7 @@ Additional Use Grant: You may make production use of the Licensed Work, provided
                       2) use the Licensed Work for use in a governmental or any
                       for-profit entity with more than fifty (50) users.
 
-Change Date:  Sep 17, 2028
+Change Date:  Sep 25, 2028
```

Comment:
How is that date chosen ?

Reply:
The script `releaser.py` update the license files:

https://github.com/Scille/parsec-cloud/blob/9cae172727d64e29e93bb77703eb48386bb2b7fc/misc/releaser.py#L588

Reply:
Thanks !